fixtures/mega-lite/mw1.json

```diff
@@ -0,0 +1,521 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "MW1",
+  "categories": [
+    "Moving Head",
+    "Color Changer"
+  ],
+  "meta": {
+    "authors": ["Ryan Goodwin"],
+    "createDate": "2024-10-06",
+    "lastModifyDate": "2024-10-06"
+  },
+  "links": {
+    "manual": [
+      "https://www.mega.lighting/_files/ugd/21ff82_e921a48d91974862860f4e5f05ed7f30.pdf"
+    ],
+    "productPage": [
+      "https://www.mega.lighting/mw1"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=StuUWiJoji8",
+      "https://www.youtube.com/watch?v=pS-_VBK4j8k"
+    ]
+  },
+  "physical": {
+    "dimensions": [323.85, 228.6, 393.7],
+    "weight": 8.6,
+    "power": 376,
+    "powerConnectors": {
+      "powerCON": "input and output"
+    },
+    "DMXconnector": "3-pin and 5-pin",
+    "bulb": {
+      "type": "7x Osram RGBW 40W LED"
+    },
+    "lens": {
+      "degreesMinMax": [9, 44]
+    }
+  },
+  "matrix": {
+    "pixelKeys": [
+      [
+        [null, "7", null, "2", null],
+        ["6", null, "1", null, "3"],
+        [null, "5", null, "4", null]
+      ]
+    ],
+    "pixelGroups": {
+      "All": "all"
+    }
+  },
+  "templateChannels": {
+    "Red $pixelKey": {
+      "highlightValue": 0,
```

Comment:
```suggestion
```

Is there a reason to highlight at 0?  I don't know that anyone uses this value, but it seems like an unusual choice to me.

Reply:
When I started defining the fixture, I was going to make the only dimmer and white highlight at 255. But that slipped my mind as I continuing to remember how to build out the fixture profiles. Removing the highlight is fine with me.